etl/steps/data/garden/un/2023-08-16/igme.py

```diff
@@ -36,14 +36,13 @@ def run(dest_dir: str) -> None:
         index=["country", "year"],
         values=["Observation value", "Lower bound", "Upper bound"],
         columns=["unit_of_measure", "indicator", "sex", "wealth_quintile"],
+        join_column_levels_with="-",
     )
-    tb.columns = [" - ".join(col).strip() for col in tb.columns.values]
-    tb = tb.reset_index()
     # Add some metadata to the variables. Getting the unit from the column name and inferring the number of decimal places from the unit.
     # If it contains " per " we know it is a rate and should have 1 d.p., otherwise it should be an integer.
     for col in tb.columns[2:]:
         unit = col.split("-")[1]
-        tb[col].metadata = VariableMeta(unit=unit)
+        tb[col].metadata = VariableMeta(unit=unit.lower().strip())
```

Comment:
Hmm I suppose that when you do this, you are rewriting the metadata of the variable, without adding all other fields (e.g. origins).
Why not doing
```
tb[col].metadata.unit = unit.lower().strip()
```
?